run-tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+
+readonly POCKETIC_VERSION="8.0.0"
+readonly POCKETIC_URL="https://github.com/dfinity/pocketic/releases/download/${POCKETIC_VERSION}/pocket-ic-x86_64-linux.gz"
+readonly ASSET_WASM_URL="https://github.com/dfinity/sdk/raw/master/src/distributed/assetstorage.wasm.gz"
+readonly WORKDIR="$(pwd)"
+readonly CANISTER_DIR="${WORKDIR}/canister_wasms"
+readonly POCKETIC_BIN="${WORKDIR}/pocket-ic"
+readonly CARGO_TARGET_DIR="${WORKDIR}/target/debug"
+
+log() { echo "[$(date -u +%Y-%m-%dT%H:%M:%SZ)] $*" >&2; }
+
+log "Downloading PocketIC v${POCKETIC_VERSION}"
+curl -fsSL --retry 3 --retry-delay 5 "${POCKETIC_URL}" -o pocket-ic.gz || {
+  log "Failed to download PocketIC"
+  exit 1
+}
+log "Extracting PocketIC"
```

Comment:
Add checksum verification?

Reply:
would be good, but it seems `checksum` is not published for `pocket-ic` releases (btw it seems to be missing in `ic-gateway` releases too)

Reply:
just download it and calculate manually

Reply:
done